mw-webapp/src/dataAccessLogic/GoalPreviewDAL.ts

```diff
@@ -17,4 +18,22 @@ export class GoalPreviewDAL {
     return goalsPreview;
   }
 
+  /**
+   * Create GoalPreview
+   * @return {string} Uuid of new Goal
+   */
+  public static async createGoalPreview(): Promise<string> {
+    const goalPreviewWithoutUuid: GoalDTOWithoutUuid = {
+      studentUuid: "",
+      metricUuids: [""],
+      description: "",
+      estimationTime: 0,
+      timeUnit: TimeUnit.minute,
+    };
+
+    const newGoalUuid = await GoalService.createGoalDTO(goalPreviewWithoutUuid);
```

Comment:
newGoalUuid does not look like uuid, recheck please

Reply:
Renamed